### ISSUE TITLE
Internal commands exposed by onmessage

### ISSUE THREAD

**Issue by FTWinston** [johanholmerin/worker-webrtc#3]
This project's proved remarkably helpful to me, but I'm seeing its internal commands being passed out of my worker's `onmessage` handler, passing both into and out of the worker.

Example "unexpected" messages include:

```javascript
{ command: "CALL", id: 1630965788199, msg: { ... }, name: "setRemoteDescription" }

{ command: "SET", id: 1630965788199,  msg: { ... }, rpcId: 11 }

{ command: "CALL", id: 1630965788199, rpcId: 12, msg: {…} }

{ command: "RPC_CALLBACK", msg: true, success: true, rpcId: 9 }
```

Is it by design that these are exposed? I could just ignore them, but it would be more useful for my library to show an "unexpected message" error if a downstream developer misuses it, rather than ignoring unexpected messages.

If this is a bug, I'll try to provide a minimal reproduction. If it's by design, would you be open to a PR that prevented these messages from being passed to the `onmessage` handler, or that added a property to the event to allow them to be easily identified?

**Comment by johanholmerin** [comment]
No, that shouldn't happen. Can you try this [branch](https://github.com/johanholmerin/worker-webrtc/tree/hide-messages) to see if it fixes it?

**Comment by FTWinston** [comment]
That resolves it perfectly, thank you!

**Comment by johanholmerin** [comment]
Fixed in 0.1.1
